src/app/components/shared/model-cards/card/card.component.ts

```diff
@@ -90,43 +37,43 @@ import { WithLoadingPipe } from "../../../../pipes/with-loading/with-loading.pip
     AsyncPipe,
     WithLoadingPipe,
   ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CardComponent implements OnInit {
+export class CardComponent {
   private readonly recordingApi = inject(AudioRecordingsService);
   private readonly session = inject(BawSessionService);
   private readonly licenseService = inject(LicensesService);
 
-  @Input() public model: Project | Region;
+  public readonly model = input<Project | Region>();
 
-  public hasNoAudio$: Observable<boolean>;
-  protected isOwner: boolean;
-  protected licenseText: string | undefined = undefined;
+  protected readonly license = resource({
+    params: () => ({ model: this.model() }),
+    loader: async ({ params }) => {
+      return await params.model.license;
```

Comment:
OK this took me a little while to figure out. You added a license getter on the model that returned a promise.

It's a neat trick, but given I don't think we have promise returning values almost anywhere in our models, I think we should avoid this for the moment.


Suggestion: 

Add a `getLicense` method to the `projects` service that accepts a `IdOr<Project>`. You can use a filter request with `only` to literally just get the license.

Or: just get invoke `show` on the project service. The request is likely already cached (because of the association fetcher) so it should be a noop - which is pretty much what you've done anyway (great!) but without introducing new conventions in our models.

Reply:
Regions only belong to one project

Reply:
`getProjectFor` seems like the right utility method, but it could use some love.

- it could accept a `Project` model so it just routes back to `show`
- and the Region branch should just be using `show` as well, and should not be using `siteIds`???